tap_google_play/streams.py

```diff
@@ -29,36 +29,45 @@ class ReviewsStream(GooglePlayStream):
         th.Property("repliedAt", th.DateTimeType),
         th.Property("reviewId", th.StringType),
         th.Property("appVersion", th.StringType),
+        th.Property("appId", th.StringType),
+        th.Property("developerId", th.StringType),
     ).to_dict()
 
     def get_records(self, context: dict | None) -> Iterable[dict]:
         """Return a generator of row-type dictionary objects."""
-        app_id = self.config.get("app_id")
-
-        continuation_token = None
-
         start_date = self.get_starting_replication_key_value(context)
         if start_date:
             start_date = parse(start_date)
 
-        self.logger.info("Getting reviews for %s", app_id)
-
-        while True:
-            result, continuation_token = reviews(
+        for app_id in self.config.get("app_id_list", [self.config.get("app_id")]):
+            self.logger.info("Getting reviews for %s", app_id)
+            app_details = app(
                 app_id,
                 lang="en",
                 country="us",
-                sort=Sort.NEWEST,
-                count=1000,
-                continuation_token=continuation_token,
             )
+            continuation_token = None
+            while True:
+                result, continuation_token = reviews(
+                    app_id,
+                    lang="en",
+                    country="us",
+                    sort=Sort.NEWEST,
```

Comment:
Weirdly, in Python 3.11 int enums behave in a weird way. So, let's use the raw value:

```suggestion
                    sort=Sort.NEWEST.value,
```